### ISSUE TITLE
Write manuscript version of talk

### ISSUE THREAD

**Issue by jdsteinbach** [jdsteinbach/typetuner#8]
I wrote it once and then I lost it. Committed but didn't push. I'm weeping at my foolishness.

However, if someone could transcribe: https://www.youtube.com/watch?v=_PGOlb2eKf0&feature=youtu.be, and put the results in a file `transcription.md` in this repo, that'd be AWESOME.

**Comment by Jdtanacredi** [comment]
I am starting this, will try to finish it up this week.

**Comment by jdsteinbach** [comment]
Many thanks! 👏

**Comment by murphytrueman** [comment]
Happy to assist on this if @Jdtanacredi can't get through it all :)

**Comment by jdsteinbach** [comment]
@murphytrueman If you want to check out this transcription request, that'd be great! (I'll hold off for a day or two on adding the hacktoberfest label & drawing attention to give you a chance to see if you're interested.)
https://github.com/jdsteinbach/sass-postcss/issues/1

**Comment by murphytrueman** [comment]
Hey @jdsteinbach, sounds fantastic and happy to do :) Am I ignoring the slides in the video and just transcribing the audio?

**Comment by jdsteinbach** [comment]
Yes - I typically say each bullet point aloud, and code samples don't need to be transcribed.

**Comment by murphytrueman** [comment]
Sure! Sounds good, I'll start on this shortly and get through it today. Are you happy for me to tidy it up a little so it makes more sense as a reader/remove some um's? Sorry if these are silly questions! Just unsure if you're wanting a very literal transcription or are happy for a little tidy-up. Thanks James!

**Comment by jdsteinbach** [comment]
No problem with the questions! A little edit / clean up is great.
